src/types/navigation/index.ts

```diff
@@ -129,6 +129,10 @@ export interface IPinCodeSetProps {
   headerSubTitle: string;
 }
 
+export interface ILockProps {
+  onVerificationSuccess: () => void
```

Comment:
add headerSubTitle  and headerTitle here as optional, as you can pass them in for the pin code screen

Reply:
this comment can be ignored since we set a static title and subtitle